makefile

```diff
@@ -29,6 +29,9 @@ chrome/script.js: chrome/script.browserify.js
 chrome/inject.js: chrome/inject.browserify.js
 	$(BROWSERIFY) chrome/inject.browserify.js -o chrome/inject.js
 
+chrome/injecttop.js: chrome/injecttop.browserify.js
+	$(BROWSERIFY) chrome/injecttop.browserify.js -o chrome/injecttop.js
```

Comment:
I need to read myself as well what exactly this `browserify` does, but looking at the currently present examples, we name the file `something.browserify.js` and gitignore `something.js`, and this command will generate this `something.js` during the build.

If this `browserify` action is also needed in your case, you should rename your file to `inject_totp.browserify.js` or something like that, and add a `.gitignore` entry for `inject_totp.js`.

Reply:
`browserify` is needed only if you have `require()` calls in your code. You don't, so you don't need to use browserify, and you can remove these 2 lines from `makefile` - they are not needed. Just name your file without `browserify` suffix and everything will be good.

The fact that `inject.js` is called `inject.browserify.js` and has the similar entry in `makefile` is just a mistake, it doesn't need to be declared like so, as we don't `require()` anything in `inject.js`. I'll deal with it separately.